packages/components/src/__actions__/LinkButton/v3/_docs/LinkButton.doc.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { VisuallyHidden } from "~components/VisuallyHidden"
+import { Icon } from "~components/__future__"
+import { ReversedColors } from "~components/__utilities__/v3"
+import { LinkButton } from "../index"
+
+const meta = {
+  title: "Actions/LinkButton/LinkButton (v3)",
+  component: LinkButton,
+  args: {
+    children: "Label",
+    href: "#link-button-clicked",
+  },
+  argTypes: {
+    icon: {
+      options: ["delete", "arrow", "plus"],
+      mapping: {
+        delete: <Icon isPresentational name="delete" />,
+        arrow: <Icon isPresentational name="arrow_forward" />,
+        add: <Icon isPresentational name="add" />,
+      },
+    },
+  },
+} satisfies Meta<typeof LinkButton>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {}
+
+export const LinkButtonOpensInNewTab: Story = {
+  args: {
+    children: (
+      <>
+        Label
+        <VisuallyHidden> opens a new tab</VisuallyHidden>
+      </>
+    ),
+    href: "https://www.google.com",
+    target: "_blank",
+    icon: <Icon isPresentational name="open_in_new" shouldMirrorInRTL />,
+    iconPosition: "end",
+  },
+}
+
+export const LinkButtonVariants: Story = {
+  render: args => (
+    <>
+      <LinkButton {...args} variant="primary" />
+      <LinkButton {...args} variant="secondary" />
+      <LinkButton {...args} variant="tertiary" />
+    </>
+  ),
+  decorators: [
+    Story => (
+      <div className="flex gap-8">
+        <Story />
+      </div>
+    ),
+  ],
+}
+
+export const LinkButtonVariantsReversed: Story = {
+  render: args => (
+    <ReversedColors isReversed={true}>
+      <LinkButton {...args} variant="primary" />
+      <LinkButton {...args} variant="secondary" />
+      <LinkButton {...args} variant="tertiary" />
+    </ReversedColors>
+  ),
+  parameters: {
+    reverseColors: true,
+  },
+  decorators: [
+    Story => (
+      <div className="flex gap-8 bg-purple-700 p-16">
+        <Story />
+      </div>
+    ),
+  ],
+}
```

Comment:
Pretty sure the purpose of this param is so you don't need to put `<ReversedColors>` in your render. So you should be able to extend the Variants story, then just add this param.

```suggestion
export const LinkButtonVariantsReversed: Story = {
  ...LinkButtonVariants,
  parameters: {
    reverseColors: true,
  },
}
```